### DISCUSSION TITLE
建议加入国内大模型Kimi的翻译功能

### DISCUSSION THREAD

**Discussion by bosonnn** [immersive-translate/immersive-translate#1650] [Ideas]
### 新功能 | New Feature

1. 加入国内大模型Kimi的翻译功能

### 为什么您想要添加这些功能 | Why you want to add these features

1. Kimi 可以在国内免费使用
2. Kimi 的翻译效果还不错，可以和其它收费服务相媲美

**Comment by theowenyoung** [comment]
考虑到沉浸式翻译的大量API调用，对于用户来说通过这种方式访问 ​​monnshot AI 存在很大的风险，用户的账户很可能会因为此类调用而被封禁。

我们正在考虑接入moonshot的API，但是他们的API提供的rpm只有3，与沉浸式翻译的调用频率远远不够，至少有 60 rpm 是一个最基本的要求。
